src/Controller/Backend/BackendBase.php

```diff
@@ -42,6 +42,14 @@ protected function render($template, array $context = [], array $globals = [])
             $context = ['context' => $context];
         }
 
+        /** @deprecated Deprecated since 3.0, to be removed in 4.0. */
+        $oldGlobals = $this->app['twig']->getGlobals();
+        foreach (['user', 'users', 'bolt_version'] as $key) {
+            if (!isset($context['context'][$key])) {
+                $context['context'][$key] = $oldGlobals[$key];
+            }
+        }
```

Comment:
> As I mention in the PR text, there is scope on this … I am not going to refactor the entire backend templates this close to release … what is the middle ground here? 😄

The leave them as globals, they work fine that way regardless of twig_globals flag.